kani-compiler/src/codegen_aeneas_llbc/compiler_interface.rs

```diff
@@ -111,14 +111,16 @@ impl LlbcCodegenBackend {
         for item in &items {
             match item {
                 MonoItem::Fn(instance) => {
-                    let mut fcx = Context::new(
-                        tcx,
-                        *instance,
-                        &mut ccx.translated,
-                        &mut id_map,
-                        &mut ccx.errors,
-                    );
-                    let _ = fcx.translate();
+                    if let InstanceKind::Item = instance.kind {
```

Comment:
Same comment from other PR: is there a reason why we're ignoring non-item instances?